src/sentry/static/sentry/app/views/organizationDiscover/discover.jsx

```diff
@@ -308,53 +292,90 @@ export default class OrganizationDiscover extends React.Component {
               />
             </Fieldset>
           </Sidebar>
-          <Flex flex="1" direction="column" px={3} py={3}>
-            {data && (
-              <Result
-                data={data}
-                query={query}
-                chartData={chartData}
-                chartQuery={chartQuery}
+          <Flex direction="column" flex="1">
+            <ProjectAndTimeSelector>
+              <MultipleProjectSelector
+                value={currentQuery.projects}
+                projects={this.props.organization.projects}
+                onChange={val => this.updateField('projects', val)}
+                onUpdate={this.runQuery}
+                align="left"
+                labelMargin="-5px"
               />
-            )}
-            {!data && <Intro updateQuery={this.updateFields} />}
-            <EarlyAdopterMessage />
+              <HeaderSeparator />
+              <TimeRangeSelector
+                start={currentQuery.start}
+                end={currentQuery.end}
+                onChange={(name, val) => this.updateField(name, val)}
+                onUpdate={this.runQuery}
+                relative="14d"
+                showRelative
+                showAbsolute={false}
+                align="left"
+                labelMargin="-5px"
+              />
+            </ProjectAndTimeSelector>
+            <Flex flex="1" direction="column" px={3} py={3}>
+              {data && (
+                <Result
+                  data={data}
+                  query={query}
+                  chartData={chartData}
+                  chartQuery={chartQuery}
+                />
+              )}
+              {!data && <Intro updateQuery={this.updateFields} />}
+              <EarlyAdopterMessage />
+            </Flex>
           </Flex>
         </DiscoverBody>
       </Discover>
     );
   }
 }
 
-const DiscoverHeading = styled.h2`
+const DiscoverHeader = styled.h2`
+  display: flex;
+  font-size: 19px;
+  font-weight: normal;
+  color: ${p => p.theme.gray5};
```

Comment:
Is this one a different color now? I can't perceive any difference

Reply:
Yeah it is slightly darker since it is the header, according do design we do the gradient darkest for the title and make it slightly lighter for subtitles and everything.